### PR TITLE
relu activation added to jax frontend

### PR DESCRIPTION
**Pull request by VikramxD**

### CODE REVIEW

**Comment by VikramxD** [comment]
pull request for #2718  for https://github.com/unifyai/ivy/issues/#1515

relu

1. added ivy/functional/frontends/jax/nn/non_linear_activations.py

2. implemented 'relu'
   add ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py

3. edited  ivy/functional/frontends/jax/__init__.py
4. added  ivy/functional/frontends/jax/nn/__init__.py

**Comment by VikramxD** [comment]
@jkeane508 please review the pr and tell what to change accordingly the main frontend tests are passing and all other tests are related to something in ivy tests helpers

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py [hunk 1]
```diff
@@ -193,13 +193,13 @@ def test_tensorflow_full(
 
 # mutiply
 @given(
-        dtype_and_x=helpers.dtype_and_values(
+    dtype_and_x=helpers.dtype_and_values(
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:196]: This change is not relevant to the Issue please undo :)

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py [hunk 2]
```diff
@@ -193,13 +193,13 @@ def test_tensorflow_full(
...
         available_dtypes=tuple(
             set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
         ),
         num_arrays=2,
         shared_dtype=True,
-    ), 
+    ),
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:202]: Same as above

File: ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.jax as ivy_jax
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_jax.valid_float_dtypes),
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py:10]: We should test for all data types so we should use `ivy_jax.valid_numeric_dtypes`

File: ivy/functional/frontends/jax/__init__.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
 # flake8: noqa
 from .lax import operators
+from .nn import non_linear_activations
```
Comment (jkeane508) [inline ivy/functional/frontends/jax/__init__.py:3]: We should order imports together based on package:
e.g

lax
lax
nn
nn

**Review by jkeane508 — CHANGES_REQUESTED** [review]
See changes requested, I would recommend installing a pre-commit hook to prevent lint errors such as the one documented [here](https://lets-unify.ai/ivy/contributing/0_setting_up.html#pre-commit)

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py (hunk 1 above)
Comment (VikramxD) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:196]: This is the file thats throwing the linter issue is it ok if i leave it ?

Comment (jkeane508) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:196]: Its best to undo all changes not relevant to this commit. However, looking [here](https://github.com/unifyai/ivy/runs/7712984105?check_suite_focus=true) there are in fact lint errors in the files that are being merged in this commit.

**Pull request merged**